pgjdbc/src/test/java/org/postgresql/test/jdbc2/MixedParameterMultiStatementTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2024, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc2;
+
+import org.postgresql.PGConnection;
+import org.postgresql.PGPreparedStatement;
+import org.postgresql.PGProperty;
+import org.postgresql.jdbc.PlaceholderStyle;
+import org.postgresql.test.TestUtil;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MixedParameterMultiStatementTest extends BaseTest5 {
+  @BeforeEach
+  public void setUp() throws Exception {
+    super.setUp();
+    PGProperty.REWRITE_BATCHED_INSERTS.set(props, true);
```

Comment:
`setUp` opens a connection, so adding `REWRITE_BATCHED_INSERTS` after `setUp` is a bit too late
```suggestion
    PGProperty.REWRITE_BATCHED_INSERTS.set(props, true);
    super.setUp();
```